devnet.audius.conf

```diff
@@ -0,0 +1,42 @@
+# creator-node audius.conf
+CADDY_TLS='tls internal'
+creatorNodeEndpoint=https://localhost
+delegateOwnerWallet=0x0D38e653eC28bdea5A2296fD5940aaB2D0B8875c
+delegatePrivateKey=21118f9a6de181061a2abd549511105adb4877cf9026f271092e6813b7cf58ab
+spOwnerWallet=0xb3c66e682Bf9a85F6800c769AC5A05c18C3F331d
+web3EthProviderUrl=http://host.docker.internal:8546
+
+# discovery-provider audius.conf
+audius_discprov_url=https://localhost
+audius_delegate_owner_wallet=0x73EB6d82CFB20bA669e9c178b718d770C49BB52f
+audius_delegate_private_key=d09ba371c359f10f22ccda12fd26c598c7921bda3220c9942174562bc6a36fe8
+audius_web3_eth_provider_url=http://host.docker.internal:8546
+audius_web3_host=http://host.docker.internal:8545
+audius_web3_localhost=http://host.docker.internal:8545
+audius_solana_endpoint=http://host.docker.internal:8899
+audius_eth_contracts_token='0xdcB2fC9469808630DD0744b0adf97C0003fC29B2'
+comms_dev_mode=true
+PG_MIGRATE_TEST_MODE="false"
+
+
+
+audius_discprov_dev_mode=true
+audius_solana_track_listen_count_address=testEjzEibm3nq77VQcqCCmSMx6m3KdJHuepBH1rnue
+audius_solana_rewards_manager_account=DJPzVothq58SmkpRb1ATn5ddN2Rpv1j2TcGvM3XsHf1c
+audius_eth_contracts_token=0xdcB2fC9469808630DD0744b0adf97C0003fC29B2
+audius_solana_waudio_mint=37RCjhgV1qGV2Q54EHFScdxZ22ydRMdKMtVgod47fDP3
+audius_contracts_entity_manager_address=0x254dffcd3277C0b1660F6d42EFbB754edaBAbC2B
+audius_contracts_registry=0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+audius_eth_contracts_registry=0xABbfF712977dB51f9f212B85e8A4904c818C2b63
+audius_discprov_loglevel=debug
+audius_solana_track_listen_count_address=testEjzEibm3nq77VQcqCCmSMx6m3KdJHuepBH1rnue
+audius_solana_user_bank_program_address=testHKV1B56fbvop4w6f2cTGEub9dRQ2Euta5VmqdX9
+audius_solana_anchor_data_program_id=testRaQeeuPVd4RK4ae1sHE3xcAQyFuPHBFoVwXWc4k
+audius_solana_signer_group_address=HwecRT7whduwiLhJXiWmvrhcCh8zJhwJ4B9fXniPUDyz
+audius_solana_usdc_mint=26Q7gP8UfkDzi7GMFEQxTJaNJ8D2ybCUjex58M5MLu8y
+audius_solana_anchor_admin_storage_public_key=2Vb2BiY8d3qYkHhnMdmJQqBXv6NUtSik1Bw2nRYeP46w
+audius_solana_rewards_manager_program_address=testLsJKtyABc9UXJF8JWFKf1YH4LmqCWBC42c6akPb
```

Comment:
these are the configs to match audius-compose, we should cut this down in the configuration refactor